### PR TITLE
Add some more telemetry to graph loading

### PR DESCRIPTION
**Pull request by msullivan**
Internal telemetry shows it to be a hot spot in practice and I want to
dig a little deeper.

Gate some potentially expensive logging code on this path, while I'm
at it.

### CODE REVIEW

File: mypy/build.py [hunk 1]
```diff
@@ -1016,12 +1016,15 @@ def read_deps_cache(manager: BuildManager,
...
     """A simple helper to read a JSON file with logging."""
+    t0 = time.time()
     try:
         data = manager.metastore.read(file)
     except IOError:
         manager.log(log_error + file)
         return None
-    manager.trace(log_sucess + data.rstrip())
+    manager.add_stats(metastore_read_time=time.time() - t0)
+    if manager.verbosity() >= 2:
```
Comment (ilevkivskyi) [inline mypy/build.py]: This `if` is not needed, `trace()` already has it.

Comment (msullivan) [reply mypy/build.py]: The idea is to avoid stripping and concatenating the strings

Comment (ilevkivskyi) [reply mypy/build.py]: Oh, I see! I would then add a comment about this.

**Comment by ilevkivskyi** [comment]
I am going to include this in todays pin move.

**Pull request merged**